yt_dlp/extractor/arte.py

```diff
@@ -218,6 +236,7 @@ def _real_extract(self, url):
             'alt_title': metadata.get('subtitle') and metadata.get('title'),
             'description': metadata.get('description'),
             'duration': traverse_obj(metadata, ('duration', 'seconds')),
+            'lang': metadata.get('language'),
```

Comment:
I was wondering if there is some way to translate any string to a valid and consistent language code across extractors, I did not encounter this so far and this code seems to be 'de', 'fr' etc. so maybe there is no need for it here